packages/expect/src/__tests__/__snapshots__/matchers.test.js.snap

```diff
@@ -504,7 +504,7 @@ Received:
 `;
 
 exports[`.toBeCloseTo() throws: [1.23, 1.2249999] 1`] = `
-"<dim>expect(</><red>received</><dim>).toBeCloseTo(</><green>expected, precision</><dim>)</>
+"<dim>expect(</><red>received</><dim>).toBeCloseTo(</><green>expected</><dim>)</>
```

Comment:
One of `toBeCloseTo` snapshots should have a `precision` param set.

Reply:
Yes, see 3 tests for `accepts an optional precision argument` like needles in the haystack just above this last set of 3 for `throws` however I will make a note to self that there do not seem to be any tests that assertion throws error if either arg is not number.